app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def create
```

Comment:
- [ ] Your like create action is not functioning this is because, in the like model, the association is not set up correctly. Kindly consider fixing this.

Reply:
@Kamba56 authentication is not requirement of the project

Reply:
@Tracy-miranja, yes I know authentication is not required. I am not talking about authentication here, I am talking about model association. If your check your like model, the association to user model should be `belongs_to :author, class_name: 'User'`